File: Conditional Statement/Problem 8.Digit as Word/DigitAsWord.cs
using System;
//Problem 8. Digit as Word
//Write a program that asks for a digit (0-9), and depending on the input, shows the digit as a word (in English).
//Print “not a digit” in case of invalid input.
//Use a switch statement.
class DigitAsWord
{
    static void Main()
    {
        Console.Write("Please, enter a number from 0 to 9: ");
        string number = Console.ReadLine();
        string output;
        switch (number)
        {
            case "0":
                output = "ZERO";
                break;
            case "1":
                output = "ONE";
                break;
            case "2":
                output = "TWO";
                break;
            case "3":
                output = "THREE";
                break;
            case "4":
                output = "FOUR";
                break;
            case "5":
                output = "FIVE";
                break;
            case "6":
                output = "SIX";
                break;
            case "7":
                output = "SEVEN";
                break;
            case "8":
                output = "EIGHT";
                break;
            case "9":
                output = "NINE";
                break;
            default:
                output = "Not a digit !";
                break;
        }
        Console.WriteLine( output);
    }
}

File: Javascript Fundamentals/Conditional statements/js/DigitAsWord.js
// Write a script that asks for a digit (0-9), and depending on the input, shows the digit as a word (in English).
// Print “not a digit” in case of invalid input.
// Use a switch statement.

//var num = console.prompt("Enter digit");

var num = 7;
var result = "";
switch (num) {
	case 0: result = "zero";break;
	case 1: result = "one";break;
	case 2: result = "two";break;
	case 3: result = "three";break;
	case 4: result = "four";break;
	case 5: result = "five";break;
	case 6: result = "six";break;
	case 7: result = "seven";break;
	case 8: result = "eight";break;
	case 9: result = "nine";break;
	default: result = "You have not entered a digit [0..9]"; break;
}
console.log(result);